components/button/stories/button.stories.js

```diff
@@ -136,6 +149,23 @@ const CustomButton = ({
 				hideLabel: true,
 				iconName: iconName ?? "Edit",
 			})}
+			${when(isPendingStateSupported, () =>
```

Comment:
Should this queue off the isPending variable? I'm not seeing that included here at all.

Reply:
These two examples always show the pending state, so I've done this to hide them on the Static Black story since that one doesn't support pending state. The isPending variable still works, it just shows what the other buttons look like when pending state is applied. I liked the visibility of this option for users